serving/src/main/java/feast/serving/util/FeatureRefUtil.java

```diff
@@ -16,7 +16,7 @@
  */
 package feast.serving.util;
 
-public class SpecUtil {
+public class FeatureRefUtil {
```

Comment:
Sorry to focus so much on this, but is this purely for feature sets or also for feature references?

If it's only for feature sets, then I think it should be called `FeatureSetRefUtil` and if it's for features then it should be `FeatureRefUtil`. If it's for both, then I suppose can do something like `RefUtil`.

Also, your method name is still showing `Feast` which I think should be consistent with the class name instead.